Source/Dafny/Verifier/Translator.cs

```diff
@@ -7919,6 +7924,34 @@ enum MethodTranslationKind { SpecWellformedness, Call, CoCall, Implementation, O
       }
     }
 
+    static string MethodVerboseName(string fullName, MethodTranslationKind kind) {
+      Contract.Requires(fullName != null);
+      switch (kind) {
+        case MethodTranslationKind.SpecWellformedness:
+          return fullName + " (well-formedness)";
+        case MethodTranslationKind.Call:
+          return fullName + " (call)";
+        case MethodTranslationKind.CoCall:
+          return fullName + " (co-call)";
+        case MethodTranslationKind.Implementation:
+          return fullName + " (correctness)";
+        case MethodTranslationKind.OverrideCheck:
+          return fullName + " (override check)";
```

Comment:
I would refactor this as a (static const) dictionary from MethodTranslationKind to string and then
```
return $"{fullname} ({kindDescription[kind]})"
```